faunadb-scala/src/main/scala/faunadb/values/Value.scala

```diff
@@ -102,6 +102,8 @@ object Value {
 
   /** Create a date value. */
   def apply(localdate: LocalDate): Value = DateV(localdate)
+
+  implicit def from[T](obj: T)(implicit encoder: Encoder[T]): Value = encoder.encode(obj)
```

Comment:
This method must be present in the companion object for either T or Value, or imported directly, in order to be able to convert from T to Value. Encoder's companion will not be checked because the Encoder type is not present in the signature T => Value.